src/components/login.jsx

```diff
@@ -57,21 +65,25 @@ class Login extends Component {
     const errors = this.validate();
     this.setState({ errors: errors || {} });
 
-    var tempThis = this; // Stores current value of this
-    var loginFunction = login(this.state.account.username.trim(),
-    this.state.account.password.trim());
+    var that = this; // Stores current value of this

```

Comment:
I asked Dominic to change the names of these variables to add more clarity, so could someone please re-apply his changes? Please include his comments too!